src/router/index.js

```diff
@@ -60,6 +61,11 @@ export default new Router({
       path: '*',
       name: '404',
       component: NotFoundLayout
+    },
+    {
+      path: '/wallet-not-found',
+      name: 'WalletNotFoundLayout',
+      component: WalletNotFoundLayout
```

Comment:
Wallet not found should only be shown when the user goes to /interface without an unlocked wallet. Please remove.